src/proto/mgmt/pool.proto

```diff
@@ -206,6 +207,8 @@ message PoolQueryResp {
 	uint32 total_nodes = 9; // total nodes in pool
 	uint32 version = 10; // latest pool map version
 	uint32 leader = 11; // current raft leader
+	int32 info_bit = 12; // Pool query bit set
+	string rank_set = 13; // Set of ranks enabled or disabled
```

Comment:
In general I'm in favor of keeping the data in its simplest form (e.g. a list of ranks is just a list of integers) for as long as possible, then processing it at the higher level, when we need to display it. I don't think that passing around a string is particularly useful when it's just a list of numbers.

Reply:
In fairness to @knard-intel, this was my (possibly hare-brained) suggestion. My thinking was that at Aurora scale, it's kind of inefficient to be throwing around a uint32 array of 2048 ranks (in the case where the info includes enabled ranks for a fully-striped pool). Ken added the rankset support on the engine side, so I suggested that maybe we should use it in these messages as well.

In the long run, what might be better is to add messages to represent the [d_rank_range_t/d_rank_range_list_t](https://github.com/daos-stack/daos/blob/1ffab1d20a832ff42b9d8830ebed57bafad47e35/src/include/gurt/types.h#L92) types so that we get smaller messages without losing the structure, but I think we don't need to do that just yet.

Reply:
According to the time constraint related to this ticket, it will be not possible to add messages to represent the [d_rank_range_t/d_rank_range_list_t](https://github.com/daos-stack/daos/blob/1ffab1d20a832ff42b9d8830ebed57bafad47e35/src/include/gurt/types.h#L92). However, I could define a ticket to handle this development and add the reference of this new ticket in the code.

Reply:
Fair point, let's keep it a string for now.